src/interface/src/app/navigation/navigation.component.ts

```diff
@@ -1,23 +1,48 @@
 import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 import { Observable, Subscription } from 'rxjs';
 
 import { AuthService } from '../services';
 
+export interface SidenavLink {
+  text: string;
+  href: string;
+  icon: string;
+}
+
 @Component({
   selector: 'app-navigation',
   templateUrl: './navigation.component.html',
   styleUrls: ['./navigation.component.scss']
 })
 export class NavigationComponent implements OnInit, OnDestroy {
-  @Input()
-  sidebarOpen = false;
-
+  @Input() sidebarOpen = false;
+
+  sideNavLinks: SidenavLink[] = [
+    {
+      text: "Home",
+      href: "/region",
+      icon: "home",
+    },
+    {
+      text: "Explore",
+      href: "/map",
+      icon: "explore",
+    },
+    {
+      text: "Plan",
+      href: "/plan",
+      icon: "edit_document",
```

Comment:
We could add Login/Signup to sideNavLinks and include an extra "enabled" field in the SidenavLink interface? To reduce repetition on the template

Reply:
I'm not sure how to do it with the featureFlag in place and the async isLoggedIn$. What would you suggest?

Reply:
Ah I see, yeah it's a little more annoying with the async isLoggedIn$. The feature flag check is done by checking FeatureService.isFeatureEnabled(). Feel free to keep it the way it is!

Reply:
Ok I'll keep as is! We'll likely move the login/logout to the account icon at the top right anyway.